src/ripple/rpc/handlers/AccountTx.cpp

```diff
@@ -214,7 +214,7 @@ getLedgerRange(
     if (!bValidated)
     {
         // Don't have a validated ledger range.
-        return rpcLGR_IDXS_INVALID;
+        return rpcNOT_SYNCED;
```

Comment:
Sorry, I missed this the first time around. The gRPC methods need to be changed to handle this new error. `rpcNOT_SYNCED` should be  `FAILED_PRECONDITION` and `rpcINVALID_LEDGER_RANGE` should be `INVALID_ARGUMENT`. The errors are parsed in `populateProtoResponse()`. More details about the gRPC error codes can be found here: https://grpc.github.io/grpc/core/md_doc_statuscodes.html

Reply:
We've changed the approach of this PR, but I've not yet checked it in (hopefully later today).  Behavior for gRPC is not going to change at all.  More details later...

Reply:
We need to make the same changes to gRPC as we did for the JSON API. We can forget about versioning for gRPC, but gRPC needs to handle this new error.

Reply:
Here is a commit that has the changes I am requesting: https://github.com/cjcobb23/rippled/commit/022605f24ec10a4862f92ab9d8171816b3955957